src/Common/Commands.Common/AzurePSCmdlet.cs

```diff
@@ -682,22 +682,78 @@ protected override void ProcessRecord()
             }
         }
 
+        private string _implementationBackgroundJobDescription;
+
         /// <summary>
         /// Job Name paroperty iof this cmdlet is run as a job
         /// </summary>
         public virtual string ImplementationBackgroundJobDescription
         {
             get
             {
-                string name = "Long Running Azure Operation";
-                string commandName = MyInvocation?.MyCommand?.Name;
-                if (!string.IsNullOrWhiteSpace(commandName))
+                if (_implementationBackgroundJobDescription != null)
                 {
-                    name = string.Format("Long Running Operation for '{0}'", commandName);
+                    return _implementationBackgroundJobDescription;
                 }
+                else
+                {
+                    string name = "Long Running Azure Operation";
+                    string commandName = MyInvocation?.MyCommand?.Name;
+                    string objectName = null;
+                    if (MyInvocation?.BoundParameters?.ContainsKey("Name") == true)
```

Comment:
@maddieclayton nit: we should use the new [`IsParameterBound`](https://github.com/Azure/azure-powershell/blob/preview/src/Common/Commands.Common/Extensions/CmdletExtensions.cs#L358-L362) function to check if a parameter has been set